RELEASE_NOTES.md

```diff
@@ -72,11 +71,16 @@ v1.0.0-alpha.xx
   exceptional conveniences.
```

Comment:
[consider][minor] Could fix this one whilst we're at it "single exception type for exceptional type"

Reply:
Hows [this](https://github.com/OpenAssetIO/OpenAssetIO/pull/1157/commits/7f5d0ea833782d5883600d04bc3e9f06d80d89da)?

Reply:
Perfect.